### ISSUE TITLE
No designer Toolkit shown

### ISSUE THREAD

**Issue by fanoI** [picoe/Eto#454]
I was trying to use Eto.Forms but it seems the designer is incomplete: the toolKit is empty!

I need to write the XML by hands?

**Comment by cwensley** [comment]
The designer is not drag 'n drop (yet). Yes, you need to write the Xaml by hand.

**Comment by cwensley** [comment]
it does give you autocomplete when writing xaml, so it makes it very easy (;

**Comment by fanoI** [comment]
Thanks for the fast response drag & drop will be added in the future?

**Comment by cwensley** [comment]
Yes I would like to add drag 'n drop to the designer.  When it gets done depends on available time, other important issues, and how many people really need/want this feature.  If anyone wants to step up to the plate and help out with this, it would be more than welcome too. (;

**Comment by ris-work** [comment]
Related: #20 
https://xaml.io
